.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-    - '8.11'
+    - 10.16.3
```

Comment:
I'd like to at least allow patch updates, which can include important security updates, so could this be `'10.16'` (I think the quotes are necessary for YAML to parse it as a string instead of a float if there's only one decimal point)